src/ruling.rs

```diff
@@ -115,12 +115,12 @@ impl Ruling {
     /// # Examples
     /// ```rust
     /// use scryfall::ruling::Ruling;
-    /// match Ruling::set_and_number("ima", 65).next() {
+    /// match Ruling::set_and_number("bfz", 17).next() {
     ///     Some(rulings) => assert_eq!(
     ///         rulings
     ///             .unwrap()
     ///             .iter()
-    ///             .filter(|r| r.comment == "If the target spell is an illegal target when Mana Drain tries to resolve, it won’t resolve and none of its effects will happen. You won’t get any mana.")
+    ///             .filter(|r| r.comment == "Yes, your opponent can’t even. We know.")
```

Comment:
Why did you change this? It's a much funnier ruling I will admit but was there another reason?

Reply:
The ruling for Mana Drain has been reworded so it's and longer and not as funny. I found a short funny one instead of pasting in the new wording. Should I change it back?

Reply:
Nah it's fine. Shame they changed mana drain's